ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/generator/FixedListGenerator.java

```diff
@@ -21,13 +26,24 @@ public static void generate(File root, Map<Class, Integer> types) {
       if (c.isEnum()) {
         List<Map<String, Object>> fields = Lists.newArrayList();
 
+        FixedList annotation = (FixedList) c.getAnnotation(FixedList.class);
```

Comment:
Is this requiring all enums to have an annotation to be exported? Could you instead exclude individual types with an ignore annotation?